Tests/AnyLintTests/StatisticsTests.swift

```diff
@@ -63,28 +64,29 @@
         TestHelper.shared.reset()
 
         let checkInfo1 = CheckInfo(id: "id1", hint: "hint1", severity: .info)
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo1)],
-            in: checkInfo1
-        )
+        Statistics.shared.found(violations: [checkInfo1: [Violation(checkInfo: checkInfo1)]])
 
         let checkInfo2 = CheckInfo(id: "id2", hint: "hint2", severity: .warning)
         Statistics.shared.found(
             violations: [
-                Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Harry.swift"),
-                Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Albus.swift"),
-            ],
-            in: CheckInfo(id: "id2", hint: "hint2", severity: .warning)
+                CheckInfo(id: "id2", hint: "hint2", severity: .warning):
+                    [
+                        Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Harry.swift"),
+                        Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Albus.swift"),
+                    ]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)